DualView/app/src/main/java/com/microsoft/device/display/samples/dualview/MapView.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *  *
+ *
+ */
+
+package com.microsoft.device.display.samples.dualview
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.gestures.zoomable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.gesture.DragObserver
+import androidx.compose.ui.gesture.rawDragGestureFilter
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.res.imageResource
+import com.microsoft.device.display.samples.dualview.models.AppStateViewModel
+import com.microsoft.device.display.samples.dualview.models.restaurants
+
+@Composable
+fun MapView(modifier: Modifier, appStateViewModel: AppStateViewModel) {
+    val selectionLiveData = appStateViewModel.getSelectionLiveData()
+    val selectedIndex = selectionLiveData.observeAsState(initial = -1).value
```

Comment:
The same here with the `-1`, better to have it in a constant so we know what it is and we can reuse it (as it is used after that).